app/assets/stylesheets/components/dashboard/_content-none.scss

```diff
@@ -0,0 +1,24 @@
+.dashboard-content-none {
+  background-color: $grey-mdlight;
```

Comment:
the background seems lighter int the mocks, but if it's a color that's not added already I wouldn't bother adding another grey.

Reply:
Yeah this is #EBEBEB (design) vs #E8E8E8 (existing colour), so I decided against adding another colour for this one

Reply:
100%, I agree